src/wechaty_puppet/schemas/room_invitation.py

```diff
@@ -0,0 +1,42 @@
+"""
+Python Wechaty - https://github.com/wechaty/python-wechaty
+
+Authors:    Huan LI (李卓桓) <https://github.com/huan>
+            Jingjing WU (吴京京) <https://github.com/wj-Mcat>
+
+2020-now @ Copyright Wechaty
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from __future__ import annotations
+
+from typing import List, Optional
+from dataclasses import dataclass
+from datetime import datetime
+
+
+@dataclass
+class RoomInvitationPayload:
```

Comment:
I saw you import a `RoomInvitationPayload` from the chatie-grpc, and then you define another `RoomInvitationPayload` here.

Did you do this on purpose?